website/blog/2022-08-12-model-timing.md

```diff
@@ -0,0 +1,294 @@
+---
+title: "Rooting out resource hogs with the Model Timing tab"
+description: "Monitoring large, complex projects can be difficult. When you're running 1,000+ models in a day, how do you know which of those consistently take the longest to run? In this article, Bennie Regenold and Barr Yaron show the benefits of the Model Timing Tab in dbt Cloud."
+slug: model-timing-tab
```

Comment:
@johnblust are there any SEO concerns with this slug? Since we also have this page: https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-model-timing-tab#:~:text=Accessed%20via%20the%20%22run%20detail,displayed%20after%20the%20run%20completes.

Reply:
To be clear, this is net-new content; we aren't changing the slug of an existing page, right?

Reply:
yuppers new content! new slug! just similar to an existing slug

Reply:
That should be fine then. It's still a very different relative path. As long as the URL is unqiue (which it is), it's fine.

If we ever want to change an existing URL, then we would need to set up a 301 redirect alongside that change.

Reply:
Awesome, thanks for your wisdom!